lua/nvim-treesitter/parsers.lua

```diff
@@ -944,6 +944,17 @@ list.julia = {
   maintainers = { "@theHamsta" },
 }
 
+list.just = {
+  install_info = {
+    url = "https://github.com/IndianBoy42/tree-sitter-just",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
+  },
+  filetype = "just",
+  maintainers = { "@Hubro" },
+  experimental = true,
```

Comment:
Why is this marked as "experimental"? These days we expect parser to be somewhat stable and usable _before_ they are added here.

Reply:
Primarily because of the large changes that have happened in the parser lately, plus the fact that the README opens with "WIP:".

I've been using it for a long time though, it works great (at least for my use cases) but I thought the experimental flag was fitting.

Should I remove it?

Reply:
I you think it's fit for use (which is a prerequisite for it being fit for inclusion), yes.

Reply:
Ping; this is blocking this PR.